src/components/link-preview/link-preview.style.ts

```diff
@@ -46,6 +47,11 @@ const StyledPreviewWrapper = styled.div<{ isLoading: boolean }>`
       }
     `}
 
+  ${StyledPreviewPlaceholder} {
```

Comment:
You've imported the wrong element FYI, this is the [one](https://github.com/Sage/carbon/blob/master/src/components/link-preview/__internal__/placeholder.component.js#L4). You'll need to add top and bottom left corner radius to the `StyledPlaceholder` element as well as it's overflowing currently (or an overflow hidden in here if you prefer)

![image](https://user-images.githubusercontent.com/44157880/228315212-a9c0f8a9-bc2c-4ce4-8062-65797be1b2f8.png)


```
const StyledPlaceHolder = styled.div`
  overflow: hidden;
  position: relative;
  height: 152px;
  min-width: 152px;
  background-color: var(--colorsUtilityMajor025);
  border-bottom-left-radius: var(--borderRadius100);
  border-top-left-radius: var(--borderRadius100);
`;
```

![image](https://user-images.githubusercontent.com/44157880/228315134-752cfa96-aa15-41d1-87bc-e329e7609c63.png)